app/api/wallet/provider/mnemonic_encrypted.ts

```diff
@@ -10,7 +10,31 @@ import { EnvironmentNetwork } from '../../../environment'
 import { getBip32Option } from '../network'
 import { WalletPersistenceData, WalletType } from '../persistence'
 
-const encryption = new PrivateKeyEncryption(new Scrypt(), numOfBytes => {
+// BIP38 default, 16k, 8, 8
+const DEFAULT_SCRYPT_N_R_P = [
+  Math.pow(2, 11),
+  4, // decide stress on ram, not to reduce, to remained strong POW
+  8 // iteration, directly stack up time (if only purely single thread)
```

Comment:
How about this instead?

```suggestion
  Math.pow(2, 11),
  8, // decide stress on ram, not to reduce, to remained strong POW
  1 // iteration, directly stack up time (if only purely single thread)
```

Default was:
```ts
  /**
   * @param {number} N the CPU/memory cost; increasing this increases the overall difficulty
   * @param {number} r the block size; increasing this increases the dependency on memory latency and bandwidth
   * @param {number} p the parallelization cost; increasing this increases the dependency on multi-processing
   */
  constructor (
    private readonly N: number = 16384,
    private readonly r: number = 8,
    private readonly p: number = 1
  ) {
  }
```